### DISCUSSION TITLE
Talk call requesting unreachable peer IP on TURN server

### DISCUSSION THREAD

**Discussion by ecker00** [nextcloud/all-in-one#3098] [Questions] [answered]
I've setup an external TURN server, and I'm pretty sure that it works (tested `turnutils_uclient` from remote). But NC Talk it's unable to connect, looking at Coturn using telnet cli I see this:

```
> psp *

    1) id=000000000000000050, user <1691440740:eyvxRmxxxxxxx>:
      realm: mydomain.com
      started 5 secs ago
      expiring in 595 secs
      client protocol TLS/TCP, relay protocol UDP
      client addr 81.167.xxx.xx:61369, server addr 10.0.2.111:5349
      relay addr 10.0.2.111:60172
      fingerprints enforced: OFF
      mobile: OFF
      TLS method: TLSv1.2
      TLS cipher: ECDHE-ECDSA-AES256-GCM-SHA384
      usage: rp=3, rb=272, sp=2, sb=224
       rate: r=0, s=0, total=0 (bytes per sec)
      peers:
          172.19.0.4

    2) id=001000000000000061, user <1691440740:eyvxRmxxxxxxx>:
      realm: mydomain.com
      started 5 secs ago
      expiring in 595 secs
      client protocol TLS/TCP, relay protocol UDP
      client addr 81.167.xxx.xxx:61368, server addr 10.0.2.111:5349
      relay addr 10.0.2.111:60188
      fingerprints enforced: OFF
      mobile: OFF
      TLS method: TLSv1.2
      TLS cipher: ECDHE-ECDSA-AES256-GCM-SHA384
      usage: rp=3, rb=272, sp=2, sb=224
       rate: r=0, s=0, total=0 (bytes per sec)
      peers:
          172.19.0.4

  Total sessions: 2
  ```

The `peers` ip `172.19.0.4` is the docker container `nextcloud/aio-talk`, how can I change this? Isn't the peers address supposed to be publicly reachable? How can I configure which peers address is being used? Or is the issue my `relay addr: 10.0.2.111` being local?

Have attempted to follow the [docs](https://nextcloud-talk.readthedocs.io/en/latest/TURN/) but don't see anything covering this issue.

**Comment by szaimen** [comment, marked as answer]
Why cant you use the default, built-in turn server?

**Reply by ecker00** [reply]
I probably should, but I had setup a TURN server for another service and thought it was an idea to use the same one. Anyway, didn't end up using it and now I don't know what the default value are to reset this. 

Do you know what I need to set to get the default TURN server back?

**Reply by szaimen** [reply]
> Do you know what I need to set to get the default TURN server back?

Simply delete your custom turn settings from the talk admin settings and restart the containers. Then should the default turn settings automatically get added back.
